reports/usage-analytics/Skuid_Usage.xml

```diff
@@ -0,0 +1,77 @@
+<skuidpage unsavedchangeswarning="yes" personalizationmode="server" showsidebar="true" showheader="true">
+	<models>
+<model id="Skuid_Usage" limit="1" query="false" createrowifnonefound="false" datasource="salesforce" sobject="Skuid_Usage__c">
+	<fields>
+		<field id="Name"/>
+		<field id="CreatedById"/>
```

Comment:
1. Having CreatedById and CreatedDate fields in the model is unnecessary -- see my comments below. These fields should be removed.
2. Can we indent these `<model>` and `<models/>` nodes? Indentation in this page is a bit wonky.